src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -58,7 +58,7 @@ public PersistentDataStore() {
     List<User> users = new ArrayList<>();
 
     // Retrieve all users from the datastore.
-    Query query = new Query("chat-users");
+    Query query = new Query("chat-users").addSort("creation_time", SortDirection.ASCENDING);
```

Comment:
Do we really need/want to sort this list for all callers of this method? WDYT about just calling sort() on the list in the code that needs the sorted version.

Reply:
Hm that brings a few questions though: 
Why would we want to load an unsorted list of users? 
How would we sort a list of type users? (users.sort() would cause an error)
Would it be best to add a new function called loadUsersOrder() and call it like so:
List<User> users = PersistentStorageAgent.getInstance().loadUsersOrder(); (and add the ordering here?)
I feel like I'm overcomplicating it and would appreciate more feedback.